### ISSUE TITLE
Bringing Vue 3 support to Vuesax next

### ISSUE THREAD

**Issue by tranthinh-coding** [lusaxweb/vuesax-next#292]
I just created a package [Vuesax-alpha](https://www.npmjs.com/package/vuesax-alpha).

This is an experimental version for `Vuesax-next` to support `Vuejs 3` + `Typescript`. 

Although the documentation is still incomplete, it will be updated soon in April. 
You can still test with the APIs of the `Vuesax-next` version. (A few apis have been changed but there will be scenes migration instructions in devtool)

**Comment by ccwebgroup** [comment]
This should be supported by the community

**Comment by stefan-ysh** [comment]
I think maybe you can try this for supporting vue3.x :
[Vuesax Frontend Components](https://lusaxweb.github.io/vuesax/)

**Comment by Abiodun-Sabitu** [comment]
Please is vuesax4 currently compactible with vue3?

**Comment by tranthinh-coding** [comment]
> Please is vuesax4 currently compactible with vue3?

yes, this is vuesax-next version, works with Vue 3 and Typescript. But still not complete.

**Comment by Abiodun-Sabitu** [comment]
Having some issues trying to set it up with vue3 and JavaScript

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Abiodun-Sabitu** [comment]
> > Please is vuesax4 currently compactible with vue3?
> 
> yes, this is vuesax-next version, works with Vue 3 and Typescript. But still not complete.

Can I use it this way without Typescript? just regular Js

**Comment by tranthinh-coding** [comment]
> > > Please is vuesax4 currently compactible with vue3?
> > 
> > 
> > yes, this is vuesax-next version, works with Vue 3 and Typescript. But still not complete.
> 
> Can I use it this way without Typescript? just regular Js

You can do it, check out the [documentation](https://github.com/vuesax-alpha/vuesax-alpha#readme).
